packages/main/src/components/protected/topbar/components/ProfileMore.jsx

```diff
@@ -11,11 +11,11 @@ import useOnClickOutside from "../hooks/useClickOutside";
 
 import { useTranslation } from "react-i18next";
 
-export const Dropdown = ({ setDropdown }) => {
+export const Dropdown = ({ setDropdown, setModal }) => {
   const { t } = useTranslation();
 
   const { toggleModalState } = useContext(ProfileContext);
-  const [modal, setModal] = useState("");
+  // const [modal, setModal] = useState("");
```

Comment:
Remove this commented out code

Reply:
Commented Codes have been remove @JulianaSau